### ISSUE TITLE
Option for some html tags to remain on same line

### ISSUE THREAD

**Issue by bcreeves** [akalongman/sublimetext-codeformatter#258]
Is there an option to force some tags to stay on same line? Like `span` or `option`?

Current:

```
<select>
    <option>
        option 1
    </option>
    <option>
        option 2
    </option>
</select>

<span>
    Inline stuff
</span>
```

Desired:

```
<select>
    <option>option 1</option>
    <option>option 2</option>
</select>

<span>Inline stuff</span>
```

**Comment by RevanProdigalKnight** [comment]
Short answer: Using the RegExp formatter, not yet. Using the BS4 formatter, unknown.

Long answer: When I started working on the RegExp formatter I made some poor design choices and they're coming back now to bite me in the a**. This feature has been requested a few times now in the last couple of months. I've been working on the implementation of an option to do exactly what you're asking for, but I haven't been able to find the time the last couple of weeks to finish it and submit a pull request.
